pkg/k8s/watchers/cilium_cidr_group_test.go

```diff
@@ -735,6 +829,17 @@ func TestCIDRGroupRefsTranslate(t *testing.T) {
 								},
 							},
 						},
+						Egress: []api.EgressRule{
```

Comment:
Nit: we should update the test case name in "with FromCidrSet and ToCIDRSet rules"